Foundation/NSSwiftRuntime.swift

```diff
@@ -81,12 +81,12 @@ internal func _CFZeroUnsafeIvars<T>(_ arg: inout T) {
 }
 
 @_cdecl("__CFSwiftGetBaseClass")
-internal func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
+public func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
     return unsafeBitCast(__NSCFType.self, to:UnsafeRawPointer.self)
 }
 
 @_cdecl("__CFInitializeSwift")
-internal func __CFInitializeSwift() {
+public func __CFInitializeSwift() {
```

Comment:
The link command is 
```
/Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch x86_64 -dynamiclib -isysroot /Applications/Xcode-beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk -L/Users/spse/Files/src/swift/swift-corelibs-foundation/DerivedData/Foundation/Build/Products/Release -F/Users/spse/Files/src/swift/swift-corelibs-foundation/DerivedData/Foundation/Build/Products/Release -filelist /Users/spse/Files/src/swift/swift-corelibs-foundation/DerivedData/Foundation/Build/Intermediates.noindex/Foundation.build/Release/SwiftFoundation.build/Objects-normal/x86_64/SwiftFoundation.LinkFileList -init ___CFInitialize -install_name @rpath/SwiftFoundation.framework/Versions/A/SwiftFoundation -Xlinker -rpath -Xlinker @executable_path/../Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -mmacosx-version-min=10.11 -Xlinker -object_path_lto -Xlinker /Users/spse/Files/src/swift/swift-corelibs-foundation/DerivedData/Foundation/Build/Intermediates.noindex/Foundation.build/Release/SwiftFoundation.build/Objects-normal/x86_64/SwiftFoundation_lto.o -fobjc-link-runtime -L/Users/spse/Library/Developer/Toolchains/swift-DEVELOPMENT-SNAPSHOT-2017-08-14-a.xctoolchain/usr/lib/swift/macosx -Xlinker -add_ast_path -Xlinker /Users/spse/Files/src/swift/swift-corelibs-foundation/DerivedData/Foundation/Build/Intermediates.noindex/Foundation.build/Release/SwiftFoundation.build/Objects-normal/x86_64/SwiftFoundation.swiftmodule -twolevel_namespace -Wl,-alias_list,CoreFoundation/Base.subproj/SymbolAliases -sectcreate __UNICODE __csbitmaps CoreFoundation/CharacterSets/CFCharacterSetBitmaps.bitmap -sectcreate __UNICODE __properties CoreFoundation/CharacterSets/CFUniCharPropertyDatabase.data -sectcreate __UNICODE __data CoreFoundation/CharacterSets/CFUnicodeData-L.mapping -segprot __UNICODE r r -lcurl.3 -lxml2 -lCoreFoundation -lz -licucore -single_module -compatibility_version 150 -current_version 1303 -Xlinker -dependency_info -Xlinker /Users/spse/Files/src/swift/swift-corelibs-foundation/DerivedData/Foundation/Build/Intermediates.noindex/Foundation.build/Release/SwiftFoundation.build/Objects-normal/x86_64/SwiftFoundation_dependency_info.dat -o /Users/spse/Files/src/swift/swift-corelibs-foundation/DerivedData/Foundation/Build/Products/Release/SwiftFoundation.framework/Versions/A/SwiftFoundation
```

There is a `-lCoreFoundation` in there, so I think you are correct, I will see if I can fix up the linker command.